messaging-experiments.json

```diff
@@ -261,5 +261,92 @@
         }
       ]
     }
+  },
+  {
+    "id": "MULTISTAGE_ABOUT_WELCOME_TEST",
+    "enabled": true,
+    "filter_expression": "env.version >= '78.' && env.version < '80.'",
+    "targeting": "localeLanguageCode == \"en\" && firefoxVersion >= 78 && (!('trailhead.firstrun.didSeeAboutWelcome'|preferenceValue) || 'multistage_aboutwelcome_test' in activeExperiments)",
```

Comment:
Simplified targeting and filter expression for easier testing in local dev when pointed to stage